nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -112,13 +114,29 @@ public Description createErrorDescription(
     return builder.build();
   }
 
-  private boolean isOptionalTypeErrorAndSuppressed(
-      ErrorMessage errorMessage, @Nullable Tree suggestTree) {
-    return errorMessage.messageType.equals(GET_ON_EMPTY_OPTIONAL)
-        && suggestTree != null
-        && ASTHelpers.getSymbol(suggestTree) != null
-        && symbolHasSuppressWarningsAnnotation(
-            ASTHelpers.getSymbol(suggestTree), OPTIONAL_CHECK_NAME);
+  private static boolean canHaveSuppressWarningsAnnotation(Tree tree) {
```

Comment:
Maybe `nodeCanHaveSuppressWarningsAnnotation` or `treeCanHaveSuppressWarningsAnnotation`? But I feel that it's clear enough from the parameter type and it's a long enough name as is, even for Java :)